apps/coscrad-frontend-e2e/src/e2e/resources/photographs/queries/photograph.index.query-flow.e2e.cy.ts

```diff
@@ -0,0 +1,298 @@
+import { AggregateType, LanguageCode, MIMEType } from '@coscrad/api-interfaces';
+import {
+    buildDummyAggregateCompositeIdentifier,
+    buildDummyUuid,
+} from '../../../../support/utilities';
+
+const ID_OFFSET = 80;
+
+const dummyPhotographTitle = 'Photograph of historic pithouse';
+
+const aggregateCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.photograph,
+    9
+);
+
+const { id: photographId } = aggregateCompositeIdentifier;
+
+const validPhotographIndexRoute = `/Resources/Photographs/`;
+
+const mediaItemCompositeIdentifier = buildDummyAggregateCompositeIdentifier(
+    AggregateType.mediaItem,
+    4
+);
+
+const contributors = {
+    creator: {
+        id: buildDummyUuid(112),
+        firstName: 'Jamethon',
+    },
+};
+
+describe('Photograph index query flow', () => {
+    // Make this a shared utility?
+    const seedDummyMediaItem = ({
```

Comment:
Feel free to make this a cypress command.

Reply:
I've made a shared function that seems to work.  I can't do a command because 1) it uses and existing command `seedDataWithCommand` and 2) it needs the import from `@coscrad/api-interfaces` which breaks the cypress commands script.  Let me know if you think we can go ahead with this.

Reply:
I avoided using api interfaces and enums and managed to put this on the Cypress command chain.